PowerEditor/installer/nativeLang/serbian.xml

```diff
@@ -1,438 +1,1828 @@
 <?xml version="1.0" encoding="utf-8" ?>
+<!--
+	Serbian localization for Notepad++
+	Modified by:
+		- Radoš Milićev (https://github.com/rammba)
+-->
 <NotepadPlus>
-	<Native-Langue name="Srpski(Serbian)" filename="serbian.xml">
+	<Native-Langue name="Srpski(Serbian)" filename="serbian.xml"  version="8.6.9">
```

Comment:
Is the version correct here, or it should be 8.7.1? I followed structure of english file.

Reply:
You added the latest translations for current master, so you can set version to 8.7.1 which will be the upcoming release. The version is a good indicator to see when the last update to the translations was made.